libraries/AP_HAL_SITL/sitl_airspeed.cpp

```diff
@@ -41,13 +41,13 @@ void SITL_State::_update_airspeed(float airspeed)
         // compute a realistic pressure report given some level of trapper air pressure in the tube and our current altitude
         // algorithm taken from https://en.wikipedia.org/wiki/Calibrated_airspeed#Calculation_from_impact_pressure
         float tube_pressure = fabsf(_sitl->arspd_fail_pressure - _barometer->get_pressure() + _sitl->arspd_fail_pitot_pressure);
-        airspeed = 340.29409348 * sqrt(5 * (pow((tube_pressure / SSL_AIR_PRESSURE + 1), 2.0/7.0) - 1.0));
+        airspeed = 340.29409348 * safe_sqrt(5 * (pow((tube_pressure / SSL_AIR_PRESSURE + 1), 2.0/7.0) - 1.0));
```

Comment:
This is SITL.  We may actually prefer things go pear shaped here rather than returning 0.